packages/dradis-ce/PKGBUILD

```
@@ -6,7 +6,7 @@ pkgver=6292.2cdcfc13
 pkgrel=1
 pkgdesc='An open source framework to enable effective information sharing.'
 groups=('blackarch' 'blackarch-recon' 'blackarch-misc')
-url='http://dradisframework.org/'
+url='https://dradis.com/'
```

```suggestion
url='https://dradis.com/ce/'
```

Since we're packaging the CE version and that https://dradisframework.org/ is redirecting to https://dradis.com/ce/ anyway, I suggest that we keep that.

---

Agreed, thank you so much for the assist. https://dradis.com/ce/ is perfect!!